Makefile

```diff
@@ -206,7 +206,7 @@ ifneq ($(CONTROLLER_GEN_VERSION), $(shell controller-gen --version | awk -F ":"
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_GEN_VERSION) ;\
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_GEN_VERSION) ;\
```

Comment:
You can remove lines 206-208 and 210. `go install` doesn't need those.

Reply:
Also, update kustomize install.

Reply:
done